client/updateCenter.go

```diff
@@ -68,20 +71,23 @@ type InstallStatesJob struct {
 	Version         string
 }
 
-func (u *UpdateCenterManager) Status() (status *UpdateCenter, err error) {
-	api := fmt.Sprintf("%s/updateCenter/api/json?pretty=false&depth=1", u.URL)
-	var (
-		req      *http.Request
-		response *http.Response
-	)
-
-	req, err = http.NewRequest("GET", api, nil)
-	if err == nil {
-		u.AuthHandle(req)
-	} else {
-		return
-	}
+type CenterPlugin struct {
+	CompatibleWithInstalledVersion bool
+	excerpt                        string
+	Installed                      InstalledPlugin
+	minimumJavaVersion             string
+	Name                           string
+	RequiredCore                   string
+	SourceId                       string
+	Title                          string
+	URL                            string
+	Version                        string
+	Wiki                           string
+}
 
+func (u *UpdateCenterManager) Status() (status *UpdateCenter, err error) {
+	req := u.commonGet("/updateCenter/api/json?pretty=false&depth=1")
+	var response *http.Response
```

Comment:
Please use the new method (RequestWithData) instead of this style.

See also https://github.com/jenkins-zh/jenkins-cli/blob/master/client/pluginManger.go#L94